SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+If you have discovered a security vulnerability in this project, please report it
+privately. **Do not disclose it as a public issue.** This gives us time to work with you
+to fix the issue before public exposure, reducing the chance that the exploit will be
+used before a patch is released.
+
+Please submit the report as a bug on
+[JIRA](https://unicode-org.atlassian.net/jira/software/c/projects/ICU/issues/). Be sure
+to set "Security level: Sensitive". This ensures the report will only be visible to the
+ICU team.
```

Comment:
```suggestion
Please submit the bug report as an [ICU Jira issue](https://icu.unicode.org/bugs).
Be sure to set "Security level: Sensitive".
This ensures the report will only be visible to the ICU team.
```

Reply:
Good catch. Semantic line breaks are better (https://sembr.org/)

Reply:
I like the improved link text.

I don't care about the line breaks: I almost prefer to *not* line-break on sentences, to make it clear that it's a flowing paragraph. (Just personal preference.)

Reply:
Merged!